Transmission.xcodeproj/project.pbxproj

```diff
@@ -520,7 +520,6 @@
 			dstPath = "";
 			dstSubfolderSpec = 10;
 			files = (
-				A24F19210A3A796800C9C145 /* Sparkle.framework in Copy Files */,
```

Comment:
This type of change makes you wonder if it's related to your issues with Sparkle framework not being found at runtime. Could be that some manual changes or merges (done elsewhere) removed the copying of the framework into the bundle...

Reply:
The Sparkle failure occurs on a unmodified checkout of current trunk, without any Xcode change.